contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -633,16 +633,20 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
   /// @notice Returns all capabilities. This operation will copy capabilities
   /// to memory, which can be quite expensive. This is designed to mostly be
   /// used by view accessors that are queried without any gas fees.
-  /// @return bytes32[] List of hashed capability Ids
-  /// @return Capability[] List of capabilities
-  function getCapabilities() external view returns (bytes32[] memory, Capability[] memory) {
-    bytes32[] memory hashedCapabilityIds = s_hashedCapabilityIds.values();
+  /// @return hashedCapabilityIds bytes32[] List of hashed capability Ids
+  /// @return capabilities Capability[] List of capabilities
+  function getCapabilities()
+    external
+    view
+    returns (bytes32[] memory hashedCapabilityIds, Capability[] memory capabilities)
```

Comment:
This should already be in develop

Reply:
@DeividasK That doesn't include the named return vars, does it? That's what I added here